packages/amplify-provider-awscloudformation/lib/configuration-manager.js

```diff
@@ -438,6 +438,7 @@ function getCurrentConfig(context) {
         configLevel: 'general',
         config: {},
       };
+      fs.removeSync(configInfoFilePath);
```

Comment:
We don't need to remove this. Its not a single config now..there would be configs for several environments now. And it would cause problems (in other environments) if we remove the whole file.